lib/models/database/post/post_data.dart

```diff
@@ -19,12 +19,16 @@ class PostData {
   final int voteScore;
   static const String voteScoreField = "voteScore";
 
+  final int commentCount;
```

Comment:
How do we handle this change when we will add a comment to a post that does not have this field in the database ?

Is the call to `set` that automatically adds the field on the document on the database ?

Reply:
Sorry, instead of `set` I meant `increment`.

Reply:
That is a really good point indeed !
After some testing, it effectively does add automatically the field if it is missing.

I have added a test for that scenario in bcbe1b262f1c37a83bb573c27febe4ece883bf69.

Reply:
Awesome :)